leasing/report/invoice/invoicing_review.py

```diff
@@ -314,6 +332,9 @@ def get_incorrect_rent_shares_data(self, cursor):
 
     def get_incorrect_management_shares_data(self, cursor):
         today = datetime.date.today()
+        service_unit_filter = ""
+        # if input_data["service_unit"]:
```

Comment:
Why this is still commented code?

Reply:
I don't know how to get the input data from the parameters that are given in the function, or what parameter I should add to the function to get the input data where I could get the given service unit id.